sympy/solvers/ode/tests/test_systems.py

```diff
@@ -1273,7 +1269,18 @@ def test_sysode_linear_neq_order1_type2():
     assert dsolve(eq26) == sol26
     assert checksysodesol(eq26 , sol26) == (True , [0,0])
 
+    # Test Case added for issue #22715
+    # https://github.com/sympy/sympy/issues/22715
 
+    eqn1 = Eq(diff(x(t),t),-1*y(t)+10)
+    eqn2 = Eq(diff(y(t),t),5*x(t)-2*y(t)+3)
+    eq27 = [eqn1, eqn2]
```

Comment:
We don't need to make two separate equations and then put them in a list. Look at the code for all of the other examples in this function,